### ISSUE TITLE
Modal: Accessibility for X button

### ISSUE THREAD

**Issue by daneriseb** [sebgroup/green#706]
The X button in modals is currently not visible for screen readers.

I checked this with Rey. If a modal has a close button at the bottom the X button can be hidden from screen readers. Otherwise screen readers will see two close buttons.

**Comment by EldRoos** [comment]
Fixed!
